.cursor/commands/ppt.md

```
@@ -0,0 +1,461 @@
+  // 핵심 부분만 간결하게
+  oldCode();
+  ```
+
+  **변경 후:**
+  ```language
+  // 변경된 부분 강조
+  newCode(); // 개선사항 주석
+  ```
+
+❌ 나쁜 예:
+  - 100줄 이상의 전체 파일 붙여넣기
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)